h2o-r/h2o-package/R/frame.R

```diff
@@ -3530,6 +3530,8 @@ as.data.frame.H2OFrame <- function(x, ...) {
   # Versions of R prior to 3.1 should not use hex string.
   # Versions of R including 3.1 and later should use hex string.
   useHexString <- getRversion() >= "3.1"
+  # We cannot use data.table by default since its handling of escaping inside quoted csv values is not very good
+  # in some edge cases its simply impossible to load data in correct format without additional post processing
```

Comment:
What specifically do you mean by "not very good"? This is unhelpful to write in this way. Tell me what's wrong and I'll fix it. Probably already known somewhere and I'll raise the priority.  We're on the same team here.

Reply:
Sorry for being so ambiguous. On second thought I should have been more to the point and link to existing issue. 
This is the issue I was reffering to:
This is the issue: https://github.com/Rdatatable/data.table/issues/1109
Here is the test that I was not able to make pass with data.table even with changes on the back-end: https://github.com/h2oai/h2o-3/pull/4265/files#diff-c87a495277fa01b186f5300c48973bf6R33